src/pages/students/RegisterStudents.jsx

```diff
@@ -1,7 +1,120 @@
+import Sidebar from '../../components/SideBar';
+import graduated from '../../assets/graduated.svg';
+import BootstrapButton from '../../components/btnBlue.jsx';
+import '../../styles/Students.css';
+import React, { useState } from 'react';
+import axios from 'axios';
 const RegisterStudents = () => {
+  const [postStudent, setPostStudent] = useState({
+    code: '',
+    name: '',
+    email: '',
+    phone: ''
+  });
+
+  const handleInputChange = (event) => {
+    const { name, value } = event.target;
+    setPostStudent((prevFormData) => ({
+      ...prevFormData,
+      [name]: value
+    }));
+  };
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    const token = localStorage.getItem('token');
+    try {
+      const response = await axios.post('http://localhost:8000/student/register', postStudent, {
```

Comment:
This axios call should be moved to external file like a service

Reply:
profe esto aun no esta listo no funciona aun